packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -110,7 +110,9 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.tableRef.setData(dataSets[x.data])}"
         >
-            <nimble-table-column-text field-name="firstName" placeholder="no value">First Name</nimble-table-column-text>
+            <nimble-table-column-text field-name="firstName" placeholder="no value">
+                <nimble-icon-user></nimble-icon-user>
```

Comment:
nit: since this is an example that clients will likely copy, can we ensure it's accessible by including a label that screen readers can access?